File: easy/calculate_distance.go
package main

import (
	"bufio"
	"fmt"
	"log"
	"math"
	"os"
)

func sqrt(a int) int {
	return int(math.Sqrt(float64(a)))
}

func main() {
	var x1, y1, x2, y2 int
	data, err := os.Open(os.Args[1])
	if err != nil {
		log.Fatal(err)
	}
	defer data.Close()
	scanner := bufio.NewScanner(data)
	for scanner.Scan() {
		fmt.Sscanf(scanner.Text(), "(%d, %d) (%d, %d)", &x1, &y1, &x2, &y2)
		x, y := x1-x2, y1-y2
		fmt.Println(sqrt(x*x + y*y))
	}
}

File: easy/calculate_distance.rb
def dist(a, b)
  Math.sqrt(a * a + b * b).to_i
end

p = /\(([-0-9]+), ([-0-9]+)\) \(([-0-9]+), ([-0-9]+)\)/
File.open(ARGV[0]).each_line do |line|
  s = line.scan(p)[0].map(&:to_i)
  puts dist(s[0] - s[2], s[1] - s[3])
end
